create_jenkins_job.py

```diff
@@ -304,8 +329,10 @@ def create_job(os_name, job_name, template_file, additional_dict):
                 },
                 'cmake_build_type': 'Debug',
                 'enable_coverage_default': 'true',
-                'build_args_default': data['build_args_default'] + ' --packages-skip qt_gui_cpp --packages-skip-by-dep qt_gui_cpp',
-                'test_args_default': data['test_args_default'] + ' --packages-skip qt_gui_cpp --packages-skip-by-dep qt_gui_cpp',
+                'build_args_default': data['build_args_default'] + ' --packages-skip qt_gui_cpp --packages-skip-by-dep qt_gui_cpp ' +
+                                      '--packages-up-to ' + quality_level_pkgs + ' ' + testing_pkgs_for_quality_level,
```

Comment:
Since this uses `--packages-up-to` can't the above lists not be reduced to not explicitly enumerate dependencies?

Reply:
It can probably can be reduced yes, the only problem I see is that would be harder to track changes in the selected set in the future. Not sure if it is worth, if you think it is I can give it a look.

Reply:
Yeah, either or is a compromise. We can just keep it as is and see if we want to change it in the future.